app/javascript/mastodon/features/audio/index.js

```diff
@@ -336,11 +329,16 @@ class Audio extends React.PureComponent {
     const AudioContext = window.AudioContext || window.webkitAudioContext;
     const context      = new AudioContext();
     const source       = context.createMediaElementSource(this.audio);
+    const gainNode     = context.createGain();
+
+    gainNode.gain.value = this.state.volume;
```

Comment:
Shouldn't this also depend on `this.state.muted`?

```suggestion
    gainNode.gain.value = this.state.muted ? 0 : this.state.volume;
```